operations/system_management_health/Access_System_Management_Health_Services.md

```diff
@@ -32,12 +32,16 @@ This procedure enables administrators to set up the service and access its compo
 
         For more information regarding the use of the Prometheus interface, see [https://prometheus.io/docs/prometheus/latest/getting\_started/](https://prometheus.io/docs/prometheus/latest/getting_started/).
 
+        Some alerts may be falsely triggering whether based on alerts which will be improved in the future or alerts related to whether all software products have been installed yet. See [Troubleshoot Prometheus Alerts](operations/system_management_health/Troubleshoot_Prometheus_Alerts.md)
```

Comment:
- Missing period in second line.
- Changing the wording to something similar to the following might make this easier to follow:

Some alerts may be falsely triggered. This occurs if they are alerts which will be improved in a future release, or if they are alerts impacted by if all software products have been installed yet.